dvc/repo/get.py

```diff
@@ -49,16 +66,30 @@ def get(url, path, out=None, rev=None):
             # the same cache file might be used a few times in a directory.
             repo.cache.local.cache_types = ["reflink", "hardlink", "copy"]
 
-            o = repo.find_out_by_relpath(path)
+            output = None
+            output_error = None
+
+            try:
+                output = repo.find_out_by_relpath(path)
+            except OutputNotFoundError as ex:
+                output_error = ex
+
+            is_git_file = output_error and not os.path.isabs(path)
+            is_not_cached = output and not output.use_cache
+
+            if is_git_file or is_not_cached:
+                return _copy_git_file(repo, path, out, url)
```

Comment:
Since it doesn't return anything
```suggestion
                _copy_git_file(repo, path, out, url)
                return
```